csatar-theme/.gitignore

```diff
@@ -0,0 +1,4 @@
+/node_modules
+/vendor
+composer.lock
+package-lock.json
```

Comment:
Lock files are intended to be commited to source control. It is the way to make sure everyone has the same packages

Reply:
1. I wanted to keep it until we create a task for Kincső, to have a them-preview file with right dimensions (if wee need it)
2. This is the way jQuery and bootstrap was included to Demo theme, I just upgraded bootstrap.
3. I remember that we talked about it, but I don't remember there was a final decision.
4. Added
5. Removed those files from .gitignore (were added by default in Demo theme)